### PR TITLE
Airdrop script

### PR DESCRIPTION
**Pull request by franckc**
### Description:

Script that takes a list of addresses as input and sends tokens.
Uses the faucet_txn table to record the transactions.

### Checklist:

- [X] Test your work and double-check to confirm that you didn't break anything
- [ ] Wrap any displayed ETH addresses with [`formattedAddress`](https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/utils/user.js#L15-L17)
- [ ] Wrap any new text/strings for translation
- [ ] Run `npm run translations` if there are any changes to translated strings
- [ ] Map any new environment variables with a default value in the Webpack config
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/origin-docs)

### CODE REVIEW

File: infra/faucet/src/scripts/airDrop.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+      const faucetTxn = await db.FaucetTxn.create({
+        campaignId: this.campaignId,
+        status: enums.FaucetTxnStatuses.Pending,
+        fromAddress: this.sender.toLowerCase(),
+        toAddress: toAddress.toLowerCase(),
+        amount: this.amount, // Amount in natural units (Wei for ETH).
+        currency: 'OGN'
+      })
+
+      if (!this.dryRun) {
```
Comment (sparrowDom) [inline infra/faucet/src/scripts/airDrop.js]: good idea doing a dry run 👍

File: infra/faucet/src/scripts/airDrop.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  return args
+}
+
+/**
+ * Reads a file having an eth address on each line
+ * @param filename
+ * @returns {string[]}
+ */
+function readRecipients(filename) {
+  const data = fs.readFileSync(filename).toString()
```
Comment (sparrowDom) [inline infra/faucet/src/scripts/airDrop.js]: not sure if `readFileSync` throws an error if the file does not exist

File: infra/faucet/src/scripts/airDrop.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+  const addresses = data
+    .split('\n')
+    .map(l => l.trim().toLowerCase())
+    .filter(l => Boolean(l))
+  const seen = {}
+  for (const address of addresses) {
+    if (!Web3.utils.isAddress(address)) {
+      throw new Error(`Invalid address: ${address}`)
+    }
+    if (seen[address]) {
```
Comment (tomlinton) [inline infra/faucet/src/scripts/airDrop.js]: Maybe standardise to lower case for the address? Could it be that the same address exists multiple times with different casing?

Comment (tomlinton) [reply infra/faucet/src/scripts/airDrop.js]: Ahh nevermind I see you have above

File: infra/faucet/src/scripts/airDrop.js (hunk 2 above)
Comment (franckc) [reply infra/faucet/src/scripts/airDrop.js]: Yes, just checked, it does throw an error...

**Pull request merged**